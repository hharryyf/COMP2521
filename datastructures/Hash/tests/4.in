i 4 hello
i 21 world
s 21
