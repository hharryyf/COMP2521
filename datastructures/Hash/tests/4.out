21 correcponds name is world
the number of comparison is 3
